### ISSUE TITLE
Process for porting latest described version into latest sonified version

### ISSUE THREAD

**Issue by terracoda** [phetsims/a11y-research#119]
I am opening this issue just to get some clarity on how to port a described version to the sonified version for interviews.

It would be nice know:
1. How long this process takes, and
2. Who to ask to do the port when a new described sonified version is needed.

In addition, it would be nice to know if there is any documentation on this process, or if perhaps documentation on the process would be helpful for the team.

**Comment by zepumph** [comment]
>How long this process takes, and

This question varies a lot. In the best case this process takes about 5 minutes for me. For example, the phet-io api for the sonfication wrapper's friction sim is the same as master, so it is easy to publish a new phet-io version of friction from master and update the sonfication wrapper to point to it. Here are a few thoughts.

1. Recently phet-io has changed, and we  need to "update" the sonification wrapper for older sims (everything but friction) to work with newly published dev releases. This process only takes a few minutes for me to do, but it needs to be done for each sim to be updated. See https://github.com/phetsims/phet-io-wrapper-sonification/issues/60 for details. 

2. On top of this, there is the potential that old phet-io apis are different that newer ones, but most often (as was the case for a GFLB version that I republished for @mikewinters10 yesterday), the sonification needs of phet-io are limited, and likely to be stable.

>Who to ask to do the port when a new described sonified version is needed.

I can do (1) easily, and can set up @jbphet to do it easily as well with minimal briefing. (2) has always been a potential problem, but I don't it has caused too many issues thus far.

I think it is important to get @jbphet involved in this discussion as well. 

Ideally republishing the phet-io sim and updating the sonification wrapper should be an easy and simple job. Sorry that there is recent muck about the phet-io api.

**Comment by jbphet** [comment]
Since I was tagged in the discussion, I thought I should chime in.  I agree with @zepumph that it's pretty much him and me doing this these days, and there isn't any documentation.  And I concur with @terracoda's observation that it's best to have ample notice.

**Comment by emily-phet** [comment]
Seems like we're shooting for 48 hours notice at least for any sim that we would like to use with description + sound. Closing, can reopen if we need to add more to this discussion.
